tests/ClientTest.php

```diff
@@ -46,6 +46,64 @@ public function testGetSupportedEndpoints()
         $this->assertContains('wikipediaSearch', $endpoints);
     }
 
+    public function testGetLastTotalResultsCount()
+    {
+
+        $g =& $this->client;
+
+        // make sure we start from scratch
+        $this->setUp();
```

Comment:
Please read on Fixtures: https://phpunit.readthedocs.io/en/7.3/fixtures.html

Reply:
Also, I think it's more readable to use `$this->client` directly. What's your toughts?

Reply:
> 
> 
> Also, I think it's more readable to use `$this->client` directly. What's your toughts?

I have only copied what was already in the code at [line 135](https://github.com/Aternus/geonames-client/blob/99f11baf551aebde84ea81414cbce77a7848ff04/tests/GeoNames/ClientTest.php#L135) ... ;-)

But if we split the test in multiple test, as suggested by your comment above, then we only reference the client once per test and a local variable doesn't make any sense ...

Reply:
> 
> 
> Please read on Fixtures: https://phpunit.readthedocs.io/en/7.3/fixtures.html

Ok, didn't realise that. Thanks for the hint. So I can safely remove the setUp()-call! 👍